shared/lib_shared_inverter.cpp

```diff
@@ -215,6 +258,53 @@ void SharedInverter::calculateACPower(const std::vector<double> powerDC_kW_in, c
 	convertOutputsToKWandScale(tempLoss, powerAC_Watts);
 }
 
+void SharedInverter::solve_kwdc_for_kwac(const double *x, double *f){
+    calculateACPower(x[0], StringV, Tdry_C);
+    f[0] = powerAC_kW - solver_AC;
+}
+
+using namespace std::placeholders;
```

Comment:
Using std::bind throughout to call the above defined functions with arguments to this function - is this supported on CentOS 7?

Reply:
battery tests passing on centos 7 so yup! bind is c++11 only